example/server-multi-dir-list.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const path = require('path')
+const Handlebars = require('handlebars')
+
+const fastify = require('fastify')({ logger: { level: 'trace' } })
+
+const template = `
+<html><body>
+
+dirs
+
+<ul>
+{{#dirs}}
+<li><a href="{{href}}">{{name}}</a></li>
+{{/dirs}}
+</ul>
+
+list
+
+<ul>
+{{#files}}
+<li><a href="{{href}}" target="_blank">{{name}}</a></li>
+{{/files}}
+</ul>
+</body></html>
+`
+const handlebarTemplate = Handlebars.compile(template)
+
+fastify
+  .register(require('..'), {
+    root: [path.join(__dirname, '/public'), path.join(__dirname, '/public2')],
+    prefixAvoidTrailingSlash: true,
+    wildcard: '**/*.css',
+    list: {
+      format: 'html',
+      names: ['index', 'index.html', 'index.htm', '/'],
+      render: (dirs, files) => handlebarTemplate({ dirs, files })
+    }
```

Comment:
can you maybe a explain with some comments why you're setting all these options? being it an example, it would benefit the readers in understanding what this is doing, as an extension to the docs basically

Reply:
Added comments to all examples 👍